config/routes.rb

```diff
@@ -1,3 +1,35 @@
+# == Route Map
```

Comment:
what's this?

Reply:
it was generated with annotate gem

Reply:
Did you install the gem and followed the usage instructions? you are missing the generated `auto_annotate_models` file